app/components/IndividualStudent.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import store from '../store.jsx'
+import Navbar from './Navbar'
+import {getStudentThunkCreator} from '../reducers/index.jsx'
+
+
+class getStudent extends Component{
+  constructor(){
+    super();
+  }
+
+  componentDidMount(){
+    console.log(this.props)
```

Comment:
Be mindful to clean up after yourself while debugging components. I see a few other `console.log`s laying around.